packages/react/src/components/Button/BaseButton.classNames.ts

```diff
@@ -43,7 +43,7 @@ export const getBaseButtonClassNames = memoizeFunction(
     const classNames = getGlobalClassNames(ButtonGlobalClassNames, theme || {});
 
     const isExpanded = expanded && !isSplit;
-    return mergeStyleSets({
+    return mergeStyleSets(styles.__shadowConfig__, {
```

Comment:
where is this property being set ? I have suspicion that this might be preventing tree shaking ?

Or is this just this line https://github.com/microsoft/fluentui/pull/30689/files#diff-ad81b0c5270b670398e69e2c5429c77eea2f5b8620290b9a43d01b11bcb0a971R169 that adds almost 5kb of JS to the bundle ?

Reply:
We talked about this offline but just wanted to add a comment here to posterity :)

The `__shadowConfig__` is set in `customizable` or `styled`. Most v8 components just the `styled` HOC but `Button` uses `customizable`. Setting the value in these two places allows us to hook into the existing customization/styling tooling nicely and keep changes to a minimum.